packages/loading-indicator/src/BlobLoader/BlobLoader.types.ts

```diff
@@ -0,0 +1,5 @@
+import { DarkModeProps } from '@leafygreen-ui/lib';
+
+export default interface SpinnerProps extends DarkModeProps {
```

Comment:
`export default interface` can lead to issues in babel. Better to define it explicitly

Reply:
Also, the interface should be something other than `SpinnerProps`

Reply:
we should probably be moving towards a future of more named exports (better auto imports, better types, fewer terminal warnings, etc.)